src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -41,7 +41,7 @@
     01 00 00 00 00
   ) // false
   .hash algorithm 0x00008004
-  .ver 4:0:3:0
+  .ver 4:0:4:0
```

Comment:
This should not be needed because of you have just bumped it in the previous commit and there is no shipped stable package with it yet.